addon/components/materialize-tabs-tab.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-tabs-tab';
+
+export default Ember.Component.extend({
+  layout: layout,
+  tagName: 'li',
+  classNames: ['tab', 'col'],
+
+  classNameBindings: ['tabWidthCss', 'isActive:active'],
+
+  didInsertElement: function () {
+    this._super(...arguments);
+    this.get('tabs').registerTab(this);
+  },
+
+  tabs: function () {
+    return this.nearestWithProperty('tabWidth');
+  }.property(),
```

Comment:
in the event that a tab is ever moved to a different tab container, this property would have to be explicitly invalidated via a `notifyPropertyChange`